README.md

```diff
@@ -1,79 +1,63 @@
-# Auth0 WordPress Plugin
+![wp-auth0](.github/header.png)
```

Comment:
I uploaded this to the CDN so that it works across any sites that might embed the readme
```suggestion
![wp-auth0](https://cdn.auth0.com/website/sdks/banners/wp-auth0-banner.png)
```